src/transformers/onnx/features.py

```diff
@@ -313,6 +314,13 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls=Data2VecTextOnnxConfig,
         ),
+        "roformer": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "sequence-classification",
+            "token-classification",
```

Comment:
It seems that `RoFormer` also includes `causal-lm`, `multiple-choice` and `question-answering`.